api/app_analytics/influxdb_wrapper.py

```diff
@@ -238,13 +261,16 @@ def get_multiple_event_list_for_feature(
     :param environment_id: an id of the environment to get usage for
     :param feature_name: the name of the feature to get usage for
     :param period: the influx time period to filter on, e.g. 30d, 7d, etc.
+      Note that "period" is tied to existing serializer logic. Prefer
+      "date_start" for new interfaces.
```

Comment:
Is it too much of a lift to change the argument to `date_start` here?

Reply:
The lift costed about 30 minutes of time refactoring the code around it. While I was at it I noticed another function that had the same param issue and way more callers so I've decided not to fix that one right now. I'll have to get to it once the timedelta refactor comes around.